package.json

```diff
@@ -46,6 +46,8 @@
     "http-status-codes": "^2.2.0",
```

Comment:
The app version is not updated but that's OK because we're not deploying this feature branch yet.

I can update the version in my forthcoming PR.